scripts/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This is a separate `eslintrc` for the tools script aimed at node instead of the browser.